src/ssl.c

```diff
@@ -16609,12 +16609,45 @@ int wolfSSL_get_server_tmp_key(const WOLFSSL* ssl, WOLFSSL_EVP_PKEY** pkey)
 
 static int sanityCheckProtoVersion(WOLFSSL_CTX* ctx)
```

Comment:
Change name of function to something useful like CheckSslMethodVersion.
Pass in method and mask rather than context - makes it clearer what is being done.
Also can be used when we implement wolfSSL_set_min_proto_version().

Reply:
Changed.